### ISSUE TITLE
Remove empty rows if no attributes present

### ISSUE THREAD

**Issue by Letme** [melexis/sphinx-traceability-extension#70]
In `item-attributes-matrix` we display also rows which do not have requested attributes. Provide a flag to not display a row if the filtered item does not contain the attribute.

Example:
```
.. item:: ITEM-ONE

     bla item 1

.. item:: ITEM-TWO
    :status: Open

    bla item 2


.. item-attributes-matrix:: Attributes for requirements
    :filter: ITEM-
    :attributes: status
    :noemptyrow:
    :nocaptions:
```

Should display:

Item | Status 
------ | --------
item-two |  Open

**Comment by SteinHeselmans** [comment]
by using a filter on attributes (as suggested in #72), I think it might be possible to remove empty rows from the item-attributes-matrix. Filtering with regex on non-empty attribute value - for the set of attributes under interest - should allow to not display items that don't have a value for the attributes...

Example is in 83a2787

@Letme could you check that in next release (to be created) with fix for #72 ?

**Comment by Letme** [comment]
Works perfectly if I add that attributename and regex (`:status: ^.+$`).
